test_psa_target.py

```diff
@@ -366,16 +381,22 @@ def _main():
 
     logging.info("Target - %s", args.mcu)
 
-    test_spec = _init_test_spec(args)
-    _build_regression_test(args, test_spec)
-    _build_compliance_test(args, test_spec)
+    if args.build and args.run:
+        logging.error(
+            "-b/--build (build only) and -r/--run (run only) can't be used together"
+        )
+
+    if not args.run:
```

Comment:
Inverse logic, I personally will find difficult to understand after couple of weeks :) 
what about.. setting a flag for build (depending on if build is passed, or both -b and -r not passed etc...)
and then 

```
if build:
     .......
     ......
```